src/icons/icons.qrc

```diff
@@ -250,137 +297,96 @@
         <file>flags/zm.svg</file>
         <file>flags/zw.svg</file>
         <file>L.gif</file>
+        <file>folder-documents.svg</file>
```

Comment:
`<file>L.gif</file>` seems out of place. You can use https://sortmylist.com/ to sort them easily.